warehouse/query-core/src/main/java/datawave/query/util/QueryStopwatch.java

```diff
@@ -45,13 +45,47 @@ private TraceStopwatch newStopwatch(String header) {
         return sw;
     }
     
-    public TraceStopwatch newStartedStopwatch(String header) {
-        TraceStopwatch sw = newStopwatch(header);
+    /**
+     * Creates a new Stopwatch whose time will not be tabulated into the total during summarization
+     *
+     * @return the newly created instance
+     */
+    private TraceStopwatch newNonSummarizedStopwatch(String header) {
+        checkNotNull(header);
+        
+        TraceStopwatch sw = new NonSummarizedStopWatch(header);
+        
+        watches.add(Maps.immutableEntry(header, sw));
+        
+        return sw;
+    }
+    
+    /**
+     * Creates a new stopwatch that will be tabulated into the total time during summarization iff nonSummarized is false
+     * 
+     * @param header
+     *            header header string to print during summarization
+     * @param nonSummarized
+     *            boolean to determine if this stopwatch's time is tabulated during summarization
+     * @return newly created instance
+     */
+    public TraceStopwatch newStartedStopwatch(String header, boolean nonSummarized) {
```

Comment:
I thought about adding a simple modifier to TraceStopWatch that made it "non summarized" so I'd be open to doing that if it is preferred. 

ala TraceStopwatch.makeNonSummarized()  if that is preferred.